src/app/app.component.ts

```diff
@@ -43,31 +51,50 @@ export class AppComponent implements OnInit {
           iconName: "home"
         },
         {
-          routerLink: "/",
+          routerLink: "/records",
           label: "Record Viewer",
           iconName: "record_viewer"
         },
         {
-          routerLink: "/",
+          routerLink: "/import-case",
           label: "Record Import",
```

Comment:
We should probably refactor the code for "Import Case" and change it to "Record Import".